### ISSUE TITLE
Batch size > 1 during testing

### ISSUE THREAD

**Issue by wangjialuu** [junyanz/pytorch-CycleGAN-and-pix2pix#1175]
Sorry to bother. I would like to set batch size >1 during testing. Current version of test.py is based on batchsize ==1. Could you tell me where to modify? 
I have done some modification to make it work but there is still some problems. I trained a pix2pix model with batch normalization and batchsize ==1. It works fine when I set batchsize ==1 during testing (using -- model pix2pix or test, both get fine results). But keeping other settings the same, when I set batchsize >1 during testing, results are not good as batchsize ==1. 
Should I 
（1）retrain a model with  batch normalization and batchsize >1, and test using same batchsize? 
（2）retrain a model with instance normalization? 
（3）or  modification I did to make multi-batchsize work during test is uncorrect?
Could you give me some advice?
Thank you so much!
